### PR TITLE
Add basic template for components

### PR DESCRIPTION
**Pull request by NagaSatyaGeeth**
Created a basic template that can be reused for every component
https://365zk.csb.app/
Capture

### CODE REVIEW

File: template/template.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+#github-icon {
+  margin: 3rem 3rem 0rem 2rem;
+}
+
+#dark-mode {
+  margin-top: 1.5rem;
+}
+
+#main-content {
+}
```
Comment (VishalPatil18) [inline template/template.css:52]: You could avoid adding empty classes.

**Review by VishalPatil18 — APPROVED** [review]
Just one suggestion if you could give a margin-top to the sidebar it would look much better.
The code looks quite clean. Great work @Artistole 👍

File: template/template.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  box-sizing: border-box;
+}
+
+#logo-container {
+  display: flex;
+}
+
+#logo {
+  width: 200px;
+  height: 100px;
```
Comment (Sudhanshusoni45) [inline template/template.css]: Suggestion: You can use rem instead of px would help in responsiveness

File: template/template.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+header {
+  display: flex;
+  position: fixed;
+  flex-direction: row;
+  justify-content: space-between;
+  box-shadow: 0.75em 0.75em 1em rgba(0, 0, 0, 0.3);
+  width: 100%;
+  background-color: white;
```
Comment (Sudhanshusoni45) [inline template/template.css]: Suggestion: You can use variables instead of hard coding colors.

File: template/template.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                <li class="side-bar-item"><a href="#">Lists</a></li>
+                <li class="side-bar-item"><a href="#">Navigation</a></li>
+                <li class="side-bar-item"><a href="#">Modal</a></li>
+                <li class="side-bar-item"><a href="#">Rating</a></li>
+            </ul>
+          </aside>
+    </div>
+    <script src="app.js"></script>
+</body>
+</html>
```
Comment (Sudhanshusoni45) [inline template/template.html:46]: You can enable add new line at the end feature in the VS code
[](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

Comment (Sudhanshusoni45) [inline template/template.html]: You can enable 'add new line at the end' feature in vs code to get rid of this error.
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

**Review by Sudhanshusoni45 — APPROVED** [review]
Added few suggestions, code looks quite neat, great work with the template.

**Pull request merged**